README.md

```diff
@@ -45,6 +45,14 @@
 | 2.     	| Tic Tac Toe         	| https://www.youtube.com/watch?v=BHh654_7Cmw  	        | <img src="https://img.shields.io/badge/-easy-success">  	|
 | 3.     	| Space Invaders       	| https://www.youtube.com/watch?v=crV6T3piwHQ          	| <img src="https://img.shields.io/badge/-medium-orange">  	|
 |        	|                      	|                                                     	|                                                         	|
+
+
+<h2>Java</h2>
+| S. No. 	| Name                 	| Link                                                	| Level                                                   	|
+|:------:	|:--------------------:	|:---------------------------------------------------:	|:-------------------------------------------------------:	|
+:heavy_plus_sign: Add tic tac toe in Java: https://www.youtube.com/watch?v=gQb3dE-y1S4 <img src="https://img.shields.io/badge/-easy-success">
```

Comment:
@innatepanda Please correct the markdown syntax here else is fine. 

It should be like this :
`
| 1. | Tic Tac Toe | https://www.youtube.com/watch?v=gQb3dE-y1S4  | <img src="https://img.shields.io/badge/-easy-success"> |
`